docs/source/delta-deletion-vectors.md

```diff
@@ -11,7 +11,7 @@ Deletion vectors are a storage optimization feature that can be enabled on <Delt
 
 ## Enable deletion vectors
 
-<Delta> 2.4 and above leverages deletion vectors to accelerate `DELETE` operations on a supported Delta table.
```

Comment:
Is it better to create a table?

<paragraph: Intro to dvs and how it improves and also mention that DV support is incrementally added with each new release of Delta lake etc.>

table
|DML Command | First available version | Enabled by default since version|